images/rabbitmq/configs/latest.apko.yaml

```diff
@@ -7,6 +7,10 @@ contents:
     - rabbitmq-server
     - wolfi-baselayout
     - bash
+    - glibc-locale-en
+    - tzdata
+    # - procps do we need this?
+    # - su-exec do we need this?
```

Comment:
I suspect these are only needed if an entrypoint script is used which we're not.  Just wanted to double check so can remove if not needed.

Reply:
Yeah, you're probably right. We can add them back + a test if they're needed.